### PR TITLE
Module 3 components

### PR DESCRIPTION
**Pull request by boryskrutskykh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,18 +3,35 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
```
Comment (Badyanchik) [inline package.json]: All the packages like @testing-library/... and @types/...
also @typescript-eslint/parser eslint eslint-config-prettier prettier jest typescript-eslin

are dev dependencies

You can read something like it
https://saturncloud.io/blog/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies-in-npm-packagejson-file/

File: package.json [hunk 2]
```diff
@@ -35,13 +52,18 @@
...
+    "@storybook/addon-onboarding": "^1.0.11",
+    "@storybook/blocks": "^7.6.17",
+    "@storybook/preset-create-react-app": "^7.6.17",
+    "@storybook/react": "^7.6.17",
+    "@storybook/react-webpack5": "^7.6.17",
+    "@storybook/test": "^7.6.17",
+    "eslint-plugin-storybook": "^0.8.0",
+    "prop-types": "^15.8.1",
+    "storybook": "^7.6.17",
+    "webpack": "^5.90.3"
```
Comment (Badyanchik) [inline package.json:67]: Do we need to install webpack manually?

**Review by Badyanchik — APPROVED** [review]

**Comment by Badyanchik** [comment]
Overall, looks good! Well-done!

File: package.json (hunk 1 above)
Comment (boryskrutskykh) [reply package.json]: Fixed

**Pull request merged**